gprofiler/utils/perf_process.py

```diff
@@ -46,6 +46,7 @@ def __init__(
         extra_args: List[str],
         processes_to_profile: Optional[List[Process]],
         switch_timeout_s: int,
+        executable_args_to_profile: Optional[list],
```

Comment:
Instead of adding a new arg - we can ensure the `extra_args` are last, and then the extra args become `[ -e, event, --, sleep, 0.5 ]`

Reply:
I prefer the current approach
but idc e86ad7b4d28bcfe1caff7eb2f0493469bce6d9e7